.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,20 @@
+name: Commit Count Check
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+      with:
+        fetch-depth: 30
+    - name: Checkout master
+      run: git fetch origin master
+    - name: create local master branch
+      run: git branch master origin/master
+    - name: Conform Action
```

Comment:
how about calling this something more specific like "Checking for more than one commit"?

Reply:
errr  yes, that was a holdover from the old project, thanks for catching